### PR TITLE
Add support for convolution translation.

### PR DESCRIPTION
**Pull request by arpith-jacob**

### CODE REVIEW

File: test/test_node.py [hunk 1]
```diff
@@ -169,6 +169,46 @@ def test_constant(self):
...
     np.testing.assert_almost_equal(output["Y"].flatten(), values)
 
+  def test_conv(self):
+    node_def = helper.make_node("Conv", ["X", "weights"],
+                                ["Y"])
+    x_shape = [1, 5, 4]
+    x = self._get_rnd(x_shape)
+    weight_shape = [5, 3, 2]
+    weights = self._get_rnd(weight_shape)
+    output = run_node(node_def, [x, weights], device="CUDA")
```
Comment (tjingrant) [inline test/test_node.py]: We might need to use `supports_device` in the backend class to prevent buildbot failure.

File: onnx_tf/backend.py [hunk 2]
```diff
@@ -453,6 +453,50 @@ def handle_constant(cls, node, input_dict):
...
+    strides = node.attrs.get("strides", None)
+    if "group" in node.attrs.keys():
+      warnings.warn("Unsupported group attribute by Tensorflow in "
+                    "Conv. This attribute will be ignored.",
+                    UserWarning)
+    if "pads" in node.attrs.keys():
+      warnings.warn("Unsupported pads attribute by Tensorflow in "
+                    "Conv operator. The SAME padding algorithm will be used.",
+                    UserWarning)
+    if "kernel_shape" in node.attrs.keys():
```
Comment (tjingrant) [inline onnx_tf/backend.py:483]: I think this correspond to the tensorflow filter parameter.

Comment (arpith-jacob) [reply onnx_tf/backend.py:483]: I thought the input parameter 'weights' corresponds to the tensorflow filter parameter??

Comment (tjingrant) [reply onnx_tf/backend.py:483]: Sorry my mistake... I read this line from the TF doc:
```a rank (N+2) filter Tensor of shape```
and thought it's the end of the line.

**Comment by tjingrant** [comment]
Merging so that I can test with more complicated models.

File: onnx_tf/backend.py [hunk 3]
```diff
@@ -453,6 +453,50 @@ def handle_constant(cls, node, input_dict):
...
+      data_format = "NCW"
+    elif x_rank == 4:
+      data_format = "NCHW"
+    in_weights = input_dict[node.inputs[1]]
+    weights_rank = len(in_weights.get_shape())
+    if transpose:
+      # Translate weights from (M x C x KH x KW) to (KH x KW X C X M)
+      perm = list(range(2, weights_rank)) + [1, 0]
+    else:
+      # Translate weights from (C x M x KH x KW) to (KH x KW X C X M)
```
Comment (tjingrant) [inline onnx_tf/backend.py:470]: Hi Arpith, I think you get this one the other way around. Normal case (non-transpose) is when we have (M x C x kH x kW).

**Pull request merged**